### DISCUSSION TITLE
Add support for previewing Youtube clips

### DISCUSSION THREAD

**Discussion by Pranav2612000** [outline/outline#6940] [Ideas]
At the moment we have support for previewing and playing Youtube videos ( using iframe ). It would be great if we could add support for Youtube clips as well.

Youtube clips are of the following format
`https://www.youtube.com/embed/<id>?clip=<clipId>&clipt=<clipTimeIdentifier>`

Users can get this id by using the "Share" button for the clip, choosing "Embed" and copying the `src` from the iframe
<img width="1497" alt="image" src="https://github.com/outline/outline/assets/20909078/4ff7643e-7450-4c3a-b31a-f1eb76e95db4">

A small caveat is that the URL is `HTML encoded` so the user may need to change `&amp;` to `&`.

cc: @clearlysid

**Comment by tommoor** [comment]
Feel free to create an issue for this.

**Reply by Pranav2612000** [reply]
Great! Thank you @tommoor
Created the issue here https://github.com/outline/outline/issues/6941
